src/components/Element/index.ts

```diff
@@ -360,10 +353,12 @@ class Element extends Events {
         // add ui reference
         this._dom.ui = this;
 
+        this._onClickEvt = this._onClick.bind(this);
```

Comment:
Unfortunately, the _onClick handlers are kind of like virtual functions and call `super`. That doesn't work if they're arrow functions that aren't stored on the `prototype`. There may be a better way to refactor them, but I left that for a future PR since I'm trying not to destabilize things with this rather large PR.